tensorflow/stream_executor/rocm/rocm_dnn.cc

```diff
@@ -2938,53 +2988,6 @@ port::Status MIOpenSupport::DoPrepareForConvolution(
   return port::Status::OK();
 }
 
-// NOTE(keveman): Temporary data layout transformation until MIOpen supports
```

Comment:
Looks like you'll have to keep this PR open until carlus' branch goes to release. Util then, we still need this transform.

Reply:
We do not need this transform, irrespective of Carlos' PR.  Don't think the comment was intended for MIOpen :)

Reply:
Wow, that's surprising. I thought this is still needed in NHWC format.